roles/bmc_fw_update/tasks/main.yml

```diff
@@ -83,11 +83,15 @@
 
 - name: Print BMC Version
   ansible.builtin.debug:
-    msg: "{{ get_bmc_facts_bmc_firmware }}"
+    msg: "{{ get_bmc_facts_all_fw_versions }}"
+
+- name: Store fw version we installed
+  ansible.builtin.set_fact:
+    got_fw_version: "{{ vars.get_bmc_facts_all_fw_versions['bmc_firmware'] }}"
```

Comment:
I think linter should ask to have role name as variable prefix, for example `bmc_fw_update_got_fw_version`, no ?

Reply:
in a separate PR we should make `'bmc_firmware'` an argument

Reply:
Yes, it will be on the next one, to support cec and bmc (they are different versions)